tests/wrap.py

```diff
@@ -34,25 +34,22 @@ def testMakeControl(self):
         config.foo = 2
         config.bar.append("baz")
         control = config.makeControl()
-        self.assert_(testLib.checkControl(control, config.foo, config.bar))
+        self.assert_(testLib.checkControl(control, config.foo, config.bar._list))
```

Comment:
I think I'd prefer it if pybind11 could accepts sequences just like a Python equivalent could accept. Python thinks that `List` quacks like a list so it's a bit annoying that pybind11 can't take it as a list. I understand that this shouldn't delay the current work. Please add a comment to the tests noting the constraint requiring use of the internal attribute to get the real list.